99-make_reports.py

```diff
@@ -66,7 +66,8 @@ def run_report(subject, session=None):
 
     rep.add_figs_to_section(figs, captions)
 
-    if op.exists(fname_trans):
+    if (op.exists(fname_trans) and
+            mne.viz.backends.renderer.backend is not None):
```

Comment:
we don't hvae a get_backend method?

Reply:
We do but it will raise an exception if it fails to detect a valid backend. Maybe this should be changed:

https://github.com/mne-tools/mne-python/blob/master/mne/viz/backends/renderer.py#L140-L141

Reply:
Opened an issue upstream:
https://github.com/mne-tools/mne-python/issues/7683